packages/models/src/lib/util/paginate-result.model.ts

```diff
@@ -2,13 +2,12 @@ export interface PaginateResult<T> {
     readonly docs: T[];
     readonly totalDocs: number;
     readonly limit: number;
-    readonly hasPrevPage: boolean;
-    readonly hasNextPage: boolean;
-    readonly page: number;
+    readonly page?: number;
     readonly totalPages: number;
-    readonly offset?: number;
-    readonly prevPage: number;
-    readonly nextPage: number;
+    readonly nextPage?: number;
+    readonly prevPage?: number;
     readonly pagingCounter: number;
-    readonly meta?: {};
+    readonly hasPrevPage: boolean;
+    readonly hasNextPage: boolean;
+    readonly meta?: any;    
```

Comment:
Some extremely minor cleanup to make this match this interface match the mongoose-side type so they're mutually-compatible. Now we don't have to rely on `any`-based duck-typing.